src/assets/icons/home/Imprint.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="288" height="256" viewBox="0 0 288 256" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
please delete this file and the others you added to this path, as they are not in the `/assets/new` folder

Reply:
Done